tests/lax_numpy_test.py

```diff
@@ -2725,6 +2725,23 @@ def np_fun(arg):
       self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @jtu.sample_product(
+    [dict(shape=shape, axis1=axis1, axis2=axis2)
+      for shape in [shape for shape in all_shapes if len(shape) >= 2]
+      for axis1 in range(-len(shape), len(shape))
+      for axis2 in range(-len(shape), len(shape))
+      if (axis1 % len(shape)) == (axis2 % len(shape))
+    ],
+    dtype=default_dtypes,
+    out_dtype=[None] + number_dtypes,
+    offset=list(range(-4, 4)),
+  )
```

Comment:
This test is far more expensive than it needs to be. We should avoid the parameterization and just tests one or two cases directly.

Reply:
Thanks! The test is now reduced to test one case directly.